src/Apps/Artist/Components/ArtistConsignButton.tsx

```diff
@@ -72,13 +78,11 @@ export const ArtistConsignButtonLarge: React.FC<ArtistConsignButtonProps &
         })
       }}
     >
-      <BorderBox p={1} width="100%">
+      <BorderBox width="100%" p={1}>
         <Flex alignItems="center" width="100%" justifyContent="space-between">
           <Flex>
-            {isInMicrofunnel && imageURL && (
-              <Box pr={1}>
-                <Image src={imageURL} width={50} height={50} />
-              </Box>
+            {(isInMicrofunnel || isTargetSupply) && imageURL && (
```

Comment:
Hrm, this duplication across file and repo is triggering me, haha. I left this comment over on Eigen:

https://github.com/artsy/eigen/pull/3157#discussion_r405533643

And I think it applies here too, even more so because there's duplication inside the same file. So I don't know what to make of this, haha. How do we typically handle this kind of cross-platform duplication?

Reply:
I'll move it into the "compute" function that's in this file 👍

Reply:
Ah, that makes sense about hoisting to MP, thanks for the insight! 👍